Updated (moved to submission folder)_Challenge 1_Railroad Employee Data_hunter

huntmaj opened the pull request.

huntmaj committed:
Hunter's challenge 1 materials

huntmaj committed:
Additional freeze post for challenge_1_instructions_hmajor

SidharthSaluja reviewed (commented):
Couple of mistakes. 
posts/challenge1_instructions.qmd was written by Sean so it should be modified. You might want to revert that changes. 
Also remove _freeze/posts/challenge1_instructions/ 

Schedule a meeting with me or slack me for help

huntmaj committed:
commit

huntmaj committed:
commit

SidharthSaluja committed:
delete challenge from post folder

huntmaj committed:
commit

Merge branch 'template' of https://github.com/huntmaj/DACSS_601_Summer2023_Sec1 into template

# Conflicts:
#	posts/challenge1_instructions.qmd

huntmaj committed:
delete freeze and add new freeze
